components/common/src/templating/package.rs

```diff
@@ -68,7 +70,7 @@ impl Env {
 #[derive(Clone, Debug, Deserialize)]
 pub struct Pkg {
     #[serde(with = "serde_string")]
-    pub ident:                   PackageIdent,
+    pub ident:                   FullyQualifiedPackageIdent,
```

Comment:
This might be a change for a later PR, but it might be nice to remove the `oritgin`, `name`, `version`, and `release` fields from this struct and replace them with accessor functions that pull the desired part from the `ident` field.

Reply:
Completely agree, but I think I will hold off for now. The methods actually exist with the `Identifiable` trait. I think a future PR could do a lot of cleanup around this trait.